index.js

```diff
@@ -320,13 +325,93 @@ module.exports = {
     return options;
   },
 
+  _shouldHandleTypeScript(config) {
+      let emberCLIBabelConfig = config['ember-cli-babel'] || {};
+      if (typeof emberCLIBabelConfig.enableTypeScriptTransforms === 'boolean') {
+        return emberCLIBabelConfig.enableTypeScriptTransforms;
+      }
```

Comment:
This is great, it allows for a really nice escape hatch when our autodetection may fail (which AFAICT shouldn't happen, but it is still nice to be prepared).

Can you add this to the documentation in the README?

Reply:
There are also people wishing to use TypeScript *without* integrated type-checking, which this option would enable (by not having to install `ember-cli-typescript` just to get TypeScript transpilation).

Reply:
I also documented the automatic behavior and incompatibility of this option with e-c-ts < 4.0.